producto/posintegrado/README.md

```diff
@@ -0,0 +1,158 @@
+- El local debe disponer con un punto de corriente exclusivo de `220v` para enchufar el POS.
+
+### Requisitos del Comercio
+
+- El comercio debe contar con una o más áreas encargadas de los siguientes puntos:
+  - Procedimientos de Contingencia ante caídas de señal Internet y energía eléctrica.
+  - Procedimiento de Respaldo y Recuperación de operaciones efectuadas en POS Integrado de Transbank.
+  - Homologación en versiones sistemas de Caja.
+  - Homologación en Sistema Operativos de Caja.
+  - Manejo de Claves Supervisora y de usuarios de Sistema Operativo.
+  - Procedimientos de Seguridad y responsabilidad en el uso de la información proporcionada por Transbank y Clientes.
+  - Capacitación y divulgación de uso de nuevos sistemas.
+  - Manuales de Sistemas.
+  - Soporte de Redes.
+
+<aside class="notice">
+Se recomienda que el Establecimiento posea la misma versión de Sistema Operativo y Software de Caja en todas sus cajas. En caso de poseer distintas versiones, será necesario efectuar un proceso de Integración y Certificación con Transbank por cada uno de ellos.
+</aside>
+
+<aside class="notice">
+Ante cualquier cambio que efectúe el Establecimiento, ya sea de Sistema Operativo o Software de Caja, podrían existir impactos en la Integración efectuada con el POS Integrado, por lo que se sugiere probar la correcta operación con POS Integrado antes de efectuar el cambio en ambiente Productivo.
+</aside>
+
+## Seguridad
+
+### Confidencialidad de la Información
+
+De acuerdo a las normativas vigentes, las transacciones con Tarjeta de Crédito y Débito requieren la
+incorporación de los siguientes elementos de seguridad en el sistema:
+La información leída en el punto de venta NO debe ser almacenada en ningún sistema.
+Para efectos de cuadratura e identificación de transacciones, se debe utilizar el NÚMERO DE OPERACION.
+
+### Tratamiento de los Tracks de la Tarjeta
+
+La información grabada en el Track I y Track II será leída sólo por los dispositivos de seguridad (POS). Estos
+dispositivos procederán a encriptar el contenido del Track I y Track II para que esta información no quede en
+claro.
+
+### Tratamiento de datos sensibles
+
+Para asegurar la confidencialidad de la información, los mensajes de las transacciones o al menos los datos
+sensibles (además del PIN) deben viajar encriptados en los distintos tramos de la conexión, tanto en el
+requerimiento como en la respuesta. Como datos sensibles se consideran: número de tarjeta, fecha
+expiración, número de cuenta y monto de la transacción (dato validado en Autenticación de Mensajes o MAC).
+
+### El modelo Master/Session KEY
+
+El método actual de administración de llaves es el llamado Máster/Session Key, en el cual los PED (Pin Entry
```

Comment:
Is there a meaningful difference between a "POS" and a "PED" in the context of the POS integrado? If not, I would refrain from introducing more terminology that doesn't serve any purpose from the point of view of the integrator or merchant.

Reply:
Have to check this with Carlos. I assumed it was talking about the external keyboard you could connect to the pos in order to get the user PIN.